walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,9 +14,44 @@
 
 package com.google.sps;
 
+import com.google.sps.TimeRange;
 import java.util.Collection;
 
 public final class FindMeetingQuery {
+  /**
+    Determines the free spaces within the working day not occupied by attended events
+    An event is considered attended if isAttended(event,request) returns true.
+    // Attended Event:     |--B--|
+    // Day     : |---------------------|
+    // Output : |----------1111111----|
```

Comment:
nit: in the ASCII picture, align the `|`-s across the `Day` and `Output` lines.

Reply:
Aligned in this commit 1ad99a4